src/main/java/bio/terra/service/resourcemanagement/ResourceService.java

```diff
@@ -272,12 +300,26 @@ public UUID initializeSnapshotProject(
    * @return project resource id
    */
   public UUID getOrCreateDatasetProject(
-      BillingProfileModel billingProfile, String projectId, GoogleRegion region)
+      BillingProfileModel billingProfile,
+      String projectId,
+      GoogleRegion region,
+      String datasetName,
+      UUID datasetId,
+      Boolean isAzure)
       throws InterruptedException {
 
+    Map<String, String> labels = new HashMap<>();
+    labels.put("dataset-name", datasetName);
+    labels.put("dataset-id", datasetId.toString());
+    labels.put("project-usage", "dataset");
+
+    if (isAzure) {
+      labels.put("is-azure", "true");
```

Comment:
How about a having a general "cloud-platform" label instead of an "is-azure" label?

Reply:
This label is only going to be used in the interim as we implement Azure.  We'll be deleting all projects with this label once Azure is complete, and then the label won't be used anymore and be would be redundant.

Reply:
Got it! Thanks for clarifying